js/deer-render.js

```diff
@@ -300,8 +307,27 @@ export default class DeerRender {
     }
 }
 
+/**
+ * Go over each listed <deer-view> marked HTMLElement and process the UI requirements to draw and render to the DOM.
+ * These views will not contain annotation information.  
+ * 
+ * Note that the resolution of this promise enforses a 200ms delay.  That is for the deerInitializer.js.  It allows
+ * initializeDeerViews to be treated as an asyncronous event before initializeDeerForms interacts with the DOM from
+ * resulting rendered <deer-view> marked HTMLElements.  We plan to streamline this process in the near future.  
+ * @param {type} config A DEER configuration from deer-config.js
+ * @return {Promise} A promise confirming all views were visited and rendered.
+ */
 export function initializeDeerViews(config) {
-    const views = document.querySelectorAll(config.VIEW)
-    Array.from(views).forEach(elem => new DeerRender(elem, config))
-    document.addEventListener(DEER.EVENTS.NEW_VIEW, e => Array.from(e.detail.set).forEach(elem => new DeerRender(elem, config)))
+    return new Promise((res) => {
```

Comment:
https://api.jquery.com/jQuery.when/

We may ultimately want something like this in our utils, but this looks like a good way to go.